src/components/organisms/AppRouter/AppRouter.jsx

```diff
@@ -73,6 +74,8 @@ const AppRouter = () => {
         <Route path="/e/:step/:venueId" component={VenueEntrancePage} />
         <Route path="/in/:venueId/admin" component={VenueAdminPage} />
         <Route path="/in/:venueId" component={VenuePage} />
+        <Route path="/pr/:venueId/:roomId" component={VenuePrivateRoom} />
```

Comment:
> Why would we add this to the main AppRouter rather than somewhere like src/pages/VenuePage/TemplateWrapper.tsx?

We need another route with `venueId` and `roomId` as params. `venueId` is used for back button navigation `roomId` is used for the private video room as url identifier.

> Why do we need :venueId in the URL slug for this?

`:venueId` is used for the back button to navigate the user out of the video room back to the venue. The only problem is that the `venueId` is going to be the venue of the creator.

1. I'm in `devaliasvenue1` and invite you for a video chat
2. You are in `devaliasvenue2` and accept my invitation
3. Both of our back buttons would navigate to `devaliasvenue1`

Ideally you should go to your previous venue `devaliasvenue2`. Maybe I should store this data in the database object as well.

> What's the purpose of the /pr URL prefix as opposed to using one of our existing /in paths/similar?

The idea is to have a separate route for the video chat since it's not related to the venue in any way. Where `/in/` is used to indicate that you are in a venue.

Reply:
> We need another route with venueId and roomId as params. venueId is used for back button navigation roomId is used for the private video room as url identifier.

See below for why I don't think we need `venueId` in the URL, as we can just store it in local state.

> Ideally you should go to your previous venue devaliasvenue2

Agreed.

> Maybe I should store this data in the database object as well.

What database object? This is likely something that we can just store in our frontend Redux state/similar. I don't think we even need to represent it in the URL tbh?

> The idea is to have a separate route for the video chat since it's not related to the venue in any way. Where /in/ is used to indicate that you are in a venue.

Mm, fair. When I was originally thinking about/planning this feature it was probably going to be implemented as a type of venue. Not sure that it needs to be, but that's the line of thinking my questions were coming from.

I'm not inherently against a different url prefix being used here. But if this isn't going to be a venue, then `VenuePrivateRoom` doesn't make sense as a component name for it.